site/_quarto.yml

```diff
@@ -72,35 +74,20 @@ website:
       contents:
         - index.qmd
 
-    - title: "Get Started"
+    - title: "About"
       contents:
-        - text: "Get Started"
-          file: guide/get-started.qmd
-        - text: "---"
-        - text: "QuickStart — 15 Minutes"
-          file: guide/quickstart.qmd
-        - text: "---"
-        - text: "DEVELOPER FRAMEWORK"
-        - guide/quickstart-try-developer-framework-with-jupyterhub.qmd
-        - guide/quickstart-try-developer-framework-with-colab.qmd
-        - guide/quickstart-try-developer-framework-in-your-own-developer-environment.qmd
+        - about/overview.qmd
         - text: "---"
-        - text: "PLATFORM UI"
-        - guide/quickstart-explore-sample-model-documentation.qmd
-        - guide/quickstart-register-your-first-model.qmd
-        - guide/quickstart-generate-documentation-for-your-model.qmd
-#        - text: "Join Our Community" 
-#         file: guide/join-community.qmd
-        - guide/next-steps.qmd
+        - text: "FEATURES"
+        - about/overview-model-documentation.qmd
+        - about/overview-model-risk-management.qmd
         - text: "---"
-        - text: "OVERVIEW"
-        - file: guide/overview.qmd
-        - guide/overview-model-documentation.qmd
-        - guide/overview-model-risk-management.qmd
-        - guide/data-privacy-policy.qmd
+        - text: "REFERENCES"
```

Comment:
DISCUSS: I was a bit rushed in my previous review but I meant to add a comment about this section — it combines several things that aren't really alike. E.g. the style guide is for contributing but the license is a legal document. They have quite different intents. I don't have a ready answer for how to handle this (other than to break up the section) but let's maybe chat a bit more when we catch up today?

Reply:
Sure thing, it was the first "group" I thought of. I'm OK with breaking it up into two different sections if you have a suggestion for the header.